subsys/mpsl/cx/thread/mpsl_cx_thread.c

```diff
@@ -207,7 +207,12 @@ static uint32_t req_grant_delay_get(void)
 
 static int32_t register_callback(mpsl_cx_cb_t cb)
 {
+	const struct device *gpiob_none = NULL;
```

Comment:
Is there a need to define these variables? Why not simply call `gpiote_irq_handler(NULL, NULL, 0);`?